src/pages/calendar.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import Calendar from '@ericz1803/react-google-calendar';
+import { css } from '@emotion/react';
+import Layout from '../components/Layout';
+
+const API_KEY =
+  process.env.GATSBY_REACT_APP_GCAL_API_KEY ||
+  'AIzaSyCBklATFMNjWUjJVZswlTmoyZh27FbaHDQ';
```

Comment:
What do we typically do for api keys? Mines locked down so i dont mind showing it here